solvers_maven_conf/maven_opensmt_pom_template.xml

```diff
@@ -0,0 +1,62 @@
+<project xmlns="http://maven.apache.org/POM/4.0.0" xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance"
+  xsi:schemaLocation="http://maven.apache.org/POM/4.0.0 http://maven.apache.org/xsd/maven-4.0.0.xsd">
+
+<!--
+This file is part of JavaSMT,
+an API wrapper for a collection of SMT solvers:
+https://github.com/sosy-lab/java-smt
+
+SPDX-FileCopyrightText: 2021 Dirk Beyer <https://www.sosy-lab.org>
+
+SPDX-License-Identifier: Apache-2.0
+-->
+
+  <modelVersion>4.0.0</modelVersion>
+  <groupId>org.sosy-lab</groupId>
+  <artifactId>javasmt-solver-opensmt</artifactId>
+  <packaging>pom</packaging>
+  <name>javasmt-solver-opensmt</name>
+  <version>${stage.revision}</version>
+  <description>SMT solver OpenSMT for use in JavaSMT</description>
+  <url>https://github.com/sosy-lab/java-smt</url>
+
+  <licenses>
+      <license>
+          <name>MIT License</name>
+          <url>https://opensource.org/licenses/MIT</url>
+      </license>
+  </licenses>
+
+  <organization>
+      <name>Software Systems Lab</name>
+      <url>https://www.sosy-lab.org/</url>
+  </organization>
+
+  <scm>
+      <url>https://github.com/sosy-lab/java-smt/</url>
+      <connection>scm:git:git://github.com/sosy-lab/java-smt.git</connection>
+      <developerConnection>scm:git:git@github.com:sosy-lab/java-smt.git</developerConnection>
+  </scm>
+
+  <developers>
+      <developer>
+          <name>Karlheinz Friedberger</name>
```

Comment:
well, please do not add people's names to files without asking them :-)

Reply:
just keep it as it is now. I will take responsibility for questions in the future.